Source/DafnyCore/AST/Modules/ModuleDefinition.cs

```diff
@@ -649,7 +649,7 @@ public class AccessibleMember {
         // the add was successful
       } else {
         // there's already something with this name
-        var yes = bindings.TryLookup(subDecl.tok.val, out var prevDecl);
+        var yes = bindings.TryLookup(subDecl.Name, out var prevDecl);
```

Comment:
Just realized that the naming of the variable could be improved.... maybe 'declFound'?